packages/kbn-eslint-config/.eslintrc.js

```diff
@@ -314,7 +314,6 @@ module.exports = {
     '@kbn/eslint/no_constructor_args_in_property_initializers': 'error',
     '@kbn/eslint/no_this_in_property_initializers': 'error',
     '@kbn/eslint/no_unsafe_console': 'error',
-    '@kbn/eslint/no_unsafe_js_yaml': 'error',
```

Comment:
This was making my IDE showing error in every files...
![image](https://github.com/user-attachments/assets/6d29077e-e3cb-464b-bfd4-b1fbccfc61ca)

Reply:
Can you try resetting your environment?  I'm not able to reproduce the issue.

This was a reset change.

Reply:
I did `yarn kbn reset && yarn kbn bootstrap`. Do you suggest another reset command?

Reply:
I removed node_modules/ and reset and now the error is gone, I'll remove this change. Sorry for the noise